src/domains/Feedback/FormBase/FormBase.tsx

```diff
@@ -59,17 +39,19 @@ export default function FormBase({
     report,
     reportType,
     submitType,
-    onSuccess,
-    onFailure,
-    townhallId,
-}: Props & DefaultProps) {
+    onSuccess = () => {},
+    onFailure = () => {},
+    townhallId = '',
```

Comment:
Avoid defining default values this way -- it causes some other hard to catch problems.  Instead just use the default props below.  If you're having type issues, then I can help with that.